server/app/services/cloud/PublicFileNameFormatter.java

```diff
@@ -14,6 +14,10 @@ public final class PublicFileNameFormatter {
    *
    * <p>If this key is changed, also check with {@link ApplicantFileNameFormatter} to verify that
    * the names don't conflict.
+   *
+   * <p>If this key is changed, you may also need to update the cloud storage bucket permissions to
+   * ensure the files are still publicly visible. For AWS, see
+   * cloud/aws/templates/aws_oidc/filestorage.tf in the cloud-deploy-infra repository.
```

Comment:
suggestion (non-blocking): Link to the line in `cloud-deploy-infra` that might need to be updated so it's super clear. Is it this line? https://github.com/civiform/cloud-deploy-infra/blob/ffcc84855b7215149b494b1e2591eed510ca30ca/cloud/aws/templates/aws_oidc/filestorage.tf#L131

Reply:
Great suggestion, done.